example/example_training_loops.py

```diff
@@ -109,14 +110,21 @@ def main():
 
     # Train the policy or value based approach
     if args["algorithm"] == "PPO":
-        ppe.ppo_train(env, agent, args)
-        ppe.evaluate_ppo_network(env, agent, args)
+        #create the record class
+        record = Record(networks={'actor':agent.actor_net, 'critic': agent.critic_net}, config={})
+        ppe.ppo_train(env, agent, record, args)
+        ppe.evaluate_ppo_network(env, agent, record, args)
+        record.save()
     elif agent.type == "policy":
-        pbe.policy_based_train(env, agent, memory, args)
-        pbe.evaluate_policy_network(env, agent, args)
+        record = Record(networks={'actor':agent.actor_net, 'critic': agent.critic_net}, config={})
+        pbe.policy_based_train(env, agent, memory, record, args)
+        pbe.evaluate_policy_network(env, agent, record, args)
+        record.save()
```

Comment:
can move the record.save to the bottom to avoid repeating it three times for each

Reply:
resolved in commit e527976